vips/vip-xxx/bscmainnet.ts

```diff
@@ -0,0 +1,152 @@
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "toggleConfigActive(string)", ethereum.CRITICAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "toggleConfigActive(string)", ethereum.FAST_TRACK_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "pause()", ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "pause()", ethereum.CRITICAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "pause()", ethereum.FAST_TRACK_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "unpause()", ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "unpause()", ethereum.CRITICAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER_ETHEREUM, "unpause()", ethereum.FAST_TRACK_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [MARKET_CAP_RISK_STEWARD_ETHEREUM, "setMaxDeltaBps(uint256)", ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [MARKET_CAP_RISK_STEWARD_ETHEREUM, "setMaxDeltaBps(uint256)", ethereum.CRITICAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [MARKET_CAP_RISK_STEWARD_ETHEREUM, "setMaxDeltaBps(uint256)", ethereum.FAST_TRACK_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+
```

Comment:
We have to accept the ownership of the `RISK_STEWARD_RECEIVER_ETHEREUM` and `MARKET_CAP_RISK_STEWARD_ETHEREUM` contracts (check it in the simulation)